BEIMA.Backend.Test/MongoService/MongoConnectorTest.cs

```diff
@@ -329,6 +329,56 @@ public void ConnectorCreated_UpdateNullDeviceType_NullReturned()
             var result = mongo.UpdateDeviceType(null);
             Assert.IsNull(result);
         }
+
+        [TestCase("name", "a12345")]
+        [TestCase("key1", 12345)]
+        [TestCase("key2", true)]
+        public void InsertDeviceType_GetFilteredDeviceTypes_DeviceInList(string key, dynamic value)
+        {
+            var mongo = MongoConnector.Instance;
+            var doc = new BsonDocument
+            {
+                { key, value }
+            };
+            //Insert device type
+            var insertResult = mongo.InsertDeviceType(doc);
+            Assume.That(insertResult, Is.Not.Null);
+            Assume.That(insertResult, Is.TypeOf(typeof(ObjectId)));
+
+            //GetFiltered
+            var filter = MongoFilterGenerator.GetEqualsFilter(key, value);
+            var list = mongo.GetFilteredDeviceTypes(filter);
+            foreach (var device in list)
```

Comment:
You have a couple variables and comments still referencing devices instead of device types.

Reply:
fixed